elements/bulbs-page/bulbs-page.js

```diff
@@ -28,6 +29,35 @@ export default class BulbsPage extends BulbsHTMLElement {
     InViewMonitor.remove(this);
   }
 
+  get dimensions () {
```

Comment:
if you want to pull this out into `'bulbs-elements/util'`, I was planning on doing that. seems this will be in a few places just to keep things dry